mender.go

```diff
@@ -327,7 +330,14 @@ func (m *mender) loadAuth() menderError {
 }
 
 func (m *mender) IsAuthorized() bool {
-	return m.authMgr.IsAuthorized()
+	if m.authMgr.IsAuthorized() {
+		log.Info("authorization data present and valid")
```

Comment:
What about this one? Should it be moved to `Debug` or not?

There is one below this one as well which may be a candidate for the same.